File: app/src/main/java/ru/e6atb/chat/OveApplication.java
package ru.e6atb.chat;

import android.app.Application;

import rs.ove.crypt.proto.NativeMst5;

public final class OveApplication extends Application {
	@Override
	public void onCreate() {
		super.onCreate();
		CrashReportStore.install(this);
		NativeMst5.initialize(this);
		try {
			NativeMst5.installCrashHandler(CrashReportStore.nativeReportPath(this));
		} catch (Exception ignored) {
		}
	}
}

File: app/src/main/kotlin/ru/e6atb/chat/OveApplication.kt
package ru.e6atb.chat

import android.app.Application
import rs.ove.crypt.proto.NativeMst5

class OveApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        GithubOtaUpdater.initialize(this)
        CrashReportStore.install(this)
        NativeMst5.initialize(this)
        runCatching { NativeMst5.installCrashHandler(CrashReportStore.nativeReportPath(this)) }
    }
}
